src/test/java/org/tarantool/MsgPackLiteDeserializeTest.java

```diff
@@ -0,0 +1,475 @@
+     *
+     * @see MsgPackLiteSerializeTest#testPackFixedStrings() ()
+     */
+    @Test
+    void testUnpackFixedStrings() throws IOException {
+        inputStream = new ByteArrayInputStream(new byte[] {
+            // latin "a"
+            (byte) 0xa1, (byte) 0x61,
+            // latin "hello"
+            (byte) 0xa5, (byte) 0x68, (byte) 0x65, (byte) 0x6c, (byte) 0x6c, (byte) 0x6f,
+            // cyrillic "привет" (12 bytes in utf-8)
+            (byte) 0xac, (byte) 0xd0, (byte) 0xbf, (byte) 0xd1, (byte) 0x80, (byte) 0xd0,
+            (byte) 0xb8, (byte) 0xd0, (byte) 0xb2, (byte) 0xd0, (byte) 0xb5, (byte) 0xd1, (byte) 0x82,
+        });
+
+        assertEquals("a", msgPackLite.unpack(inputStream));
+        assertEquals("hello", msgPackLite.unpack(inputStream));
+        assertEquals("привет", msgPackLite.unpack(inputStream));
+    }
+
+    /**
+     * Unpacks strings.
+     *
+     * @see MsgPackLiteSerializeTest#testPackStrings()
+     */
+    @Test
+    void testUnpackStrings() throws IOException {
+        byte[] oneByteLength = new byte[202];
+        Arrays.fill(oneByteLength, (byte) 'a');
+        oneByteLength[0] = (byte) 0xd9;
+        oneByteLength[1] = (byte) 0xc8;
+        inputStream = new ByteArrayInputStream(oneByteLength);
+        assertEquals(new String(oneByteLength, 2, 200, StandardCharsets.UTF_8), msgPackLite.unpack(inputStream));
+
+        byte[] twoBytesLength = new byte[30003];
+        Arrays.fill(twoBytesLength, (byte) 'b');
+        twoBytesLength[0] = (byte) 0xda;
+        twoBytesLength[1] = (byte) 0x75;
+        twoBytesLength[2] = (byte) 0x30;
+        inputStream = new ByteArrayInputStream(twoBytesLength);
+        assertEquals(new String(twoBytesLength, 3, 30000, StandardCharsets.UTF_8), msgPackLite.unpack(inputStream));
+
+        byte[] fourBytesLength = new byte[80005];
+        Arrays.fill(fourBytesLength, (byte) 'c');
+        fourBytesLength[0] = (byte) 0xdb;
+        fourBytesLength[1] = (byte) 0x00;
+        fourBytesLength[2] = (byte) 0x01;
+        fourBytesLength[3] = (byte) 0x38;
+        fourBytesLength[4] = (byte) 0x80;
+        inputStream = new ByteArrayInputStream(fourBytesLength);
+        assertEquals(new String(fourBytesLength, 5, 80000, StandardCharsets.UTF_8), msgPackLite.unpack(inputStream));
+    }
+
+    /**
+     * Unpacks binaries.
+     *
+     * @see MsgPackLiteSerializeTest#testPackBinArrays()
+     */
+    @Test
+    void testUnpackBinArrays() throws IOException {
+        byte[] oneByteLength = new byte[202];
+        oneByteLength[0] = (byte) 0xc4;
+        oneByteLength[1] = (byte) 0xc8;
+        byte b = 0;
+        for (int i = 2; i < 202; i++) {
+            oneByteLength[i] = b++;
+        }
+        byte[] firstBinary = new byte[200];
+        System.arraycopy(oneByteLength, 2, firstBinary, 0, 200);
+        inputStream = new ByteArrayInputStream(oneByteLength);
+        assertArrayEquals(firstBinary, (byte[]) msgPackLite.unpack(inputStream));
```

Comment:
Unlike other tests we don't verify a type of return value here. However I failed to find a type that may be successfully casted to `byte[]` other then `byte[]` itself. Maybe this is okay, just requires a comment. Or let's check the type explicitly.